sdk/communication/Azure.Communication.Calling.Server/src/Models/ToneInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.Calling.Server
+{
+    /// <summary>
+    /// The tone info
+    /// </summary>
+    [CodeGenModel("ToneInfo", Usage = new string[] { "input", "output" }, Formats = new string[] { "json" })]
```

Comment:
nit: instead of `new string[]`, you can just use `new []`

Reply:
This is addressed.